corehq/apps/sms/templates/sms/chat.html

```diff
@@ -25,6 +25,7 @@
         <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
+        <script src="{% static 'style/js/hq.helpers.js' %}"></script>
```

Comment:
i guess my point is that the chat window is a stand-alone page so it will be really easy for someone to add something to hq.helpers.js that would only break this page and they wouldn't notice it.  i have a feeling there is a good chance this will happen again if we create the dependency, which makes me nervous since we have one project that makes heavy use of this and they couldn't use it for half of their day yesterday.  why we can't we just use ko.applyBindings here?